content/articles/managing-seats.md

```diff
@@ -14,27 +14,27 @@ categories:
 
```

Comment:
This has been one of the tricker ones to update, I'm trying to navigate it through the lens of seat = spot in the account, team member = person, but let me know if anything is still unclear here.

Reply:
It's much clearer than before.